src/components/settings/settings.scss

```diff
@@ -167,10 +167,6 @@
   font-weight: 500;
 }
 
-.stepper-modal .pgn__modal-header {
-  border-bottom: solid 7px;
-}
```

Comment:
This class name is being used as a global style in more places than just the "Settings" page. Moved these styles to `src/index.scss` instead so its more evident it's intended global styles.